fitbenchmarking/fitbenchmarking
# Add `results_dir` command line argument

#### Description of Work
This PR changes adds the `results_dir` option in the options.ini file to be a command line argument. It also creates an `[OUTPUT]` section in the options file to put the existing `results_dir` option.

I'm assuming backwards compatibility won't need too much consideration as fitbenchmarking is still in minor versioning?

Fixes #792

#### Testing Instructions

1. Try variations of the following and make sure the relevant results directories are created and opened in a browser:
```
fitbenchmarking
```
```
fitbenchmarking -r my_results_folder/
```
```
fitbenchmarking -o examples/options_template.ini -r my_results_folder2/
```
```
fitbenchmarking -r ////invalid/folder\to/_cause_error/
```

2. Also try adding the `results_dir` option to the options file in the `[OUTPUT]` section

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
